_rules/audio-or-video-avoids-automatically-playing-audio-x0paj4.md

```diff
@@ -0,0 +1,319 @@
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G171: # Playing sounds only on user request
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Audio output
+  - Visual output
+acknowledgments:
+  authors:
+    - Carlos Duarte
+  previous_authors:
+    - Anne Thyme Nørregaard
+    - Bryn Anderson
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'title-require'
+---
+
+## Applicability
+
+This rule applies to any `audio` or `video` element for which all the following are true:
+
+- **autoplay**: the element has an `autoplay` [attribute value][] of `true`; and
+- **not muted**: the element has a `muted` [attribute value][] of `false`; and
+- **not paused**: the element has a `paused` [attribute value][] of `false`; and
+- **audio duration**: the element has a [media resource][] for which the audio output lasts more than 3 seconds.
+
+## Expectation
+
+For each test target, there is at least one [instrument][] in the same [web page][] to pause or stop the audio, or turn the audio volume off independently from the overall system volume control.
+
+## Assumptions
+
+This rule assumes that the [instrument][] to control the sound must be located in the same [web page][]. [Instruments][instrument] located on other pages can still create accessibility issues for users relying on sound to navigate (e.g. screen readers users) since the autoplaying sound will interfere with their ability to find and activate the [instrument][]. If a [instrument][] external to the [web page][] is provided, it is possible to fail this rule but still satisfy [Success Criterion 1.4.2 Audio Control][sc142].
+
+This rule assumes that the [instrument][] to control the sound must be visible and accessible in order to be effective and usable by all kinds of users. If the [instrument][] is hidden to some users, it is possible to fail this rule but still satisfy [Success Criterion 1.4.2 Audio Control][sc142].
+
+No [user style sheets](https://drafts.csswg.org/css-cascade/#cascade-origin-user) are used and no changes to the [user agent default style sheet](https://drafts.csswg.org/css-cascade/#cascade-origin-ua) are in place, otherwise the test cases might have different outcomes from the ones presented here.
+
+## Accessibility Support
+
+The native `video` and `audio` controls in several browser and assistive technology combinations are not keyboard accessible and the `video` or `audio` element itself may not be announced. Authors are recommended to use custom controls for keyboard navigation and cross browser accessibility support in general. Some major browsers no longer automatically play the 'video' unless the 'video' is muted.
```

Comment:
Is this still true? Which browsers do not have keyboard accessible controls?

Reply:
I guess you're correct. Did some tests and couldn't find one. I'll remove the note.